### PR TITLE
libwebrtc 129.6668.1.0, iOS 14.0 にあげる

### PR DESCRIPTION
**Pull request by miosakuma**
申し訳ありません、SwiftPM への設定が漏れているので再度修正をしました。
お手数をおかけしますがご確認をお願いいたします。

動作確認は実施済みです。

- [UPDATE] WebRTC m129.6668.1.0 に上げる
  - @miosakuma
- [UPDATE] システム条件の iOS を 14.0 に上げる
  - IPHONEOS_DEPLOYMENT_TARGET を 14.0 に上げる
  - SwiftPM の platforms の設定を 14.0 に上げる
  - CocoaPods の platform の設定を 14.0 に上げる
  - libwebrtc の対象バージョンに追従した
    - https://webrtc.googlesource.com/src/+/9b81d2c954128831c62d8a0657c7f955b3c02d32
 
---

This pull request includes several updates to the Sora iOS SDK, primarily focusing on upgrading the WebRTC version and updating the iOS deployment target to 14.0. The changes span multiple configuration files and documentation to ensure compatibility and consistency across the project.

### WebRTC Version Upgrade:
* Updated WebRTC version from `127.6533.1.1` to `129.6668.1.0` in `Package.swift`, `Podfile`, `Podfile.dev`, `Sora.podspec`, `README.md`, and `Sora/PackageInfo.swift`. [[1]](diffhunk://#diff-f913940c58e8744a2af1c68b909bb6383e49007e6c5a12fb03104a9006ae677eL6-R10) [[2]](diffhunk://#diff-8f7d6adf31268a2d897ee34bd170592648d6e520aa237104395e4a4438af50cbL4-R8) [[3]](diffhunk://#diff-3c49ded39aba3ab23c6fa84f595084cdcf5d219729912f355a7eb24895cbfeeaL4-R8) [[4]](diffhunk://#diff-74da782083bbd46f709bd607809c5d9f9fe4294a93679382cb722d3e22762067L11-R18) [[5]](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L3-R3) [[6]](diffhunk://#diff-c5d413f60acf7ce97a6841dc8c803506f2a5345114f051d60989b5b3c4ba66cbL12-R24)

### iOS Deployment Target Update:
* Increased the iOS deployment target from 13.0 to 14.0 in `Package.swift`, `Podfile`, `Podfile.dev`, `Sora.podspec`, and `Sora.xcodeproj/project.pbxproj`. [[1]](diffhunk://#diff-f913940c58e8744a2af1c68b909bb6383e49007e6c5a12fb03104a9006ae677eL19-R19) [[2]](diffhunk://#diff-8f7d6adf31268a2d897ee34bd170592648d6e520aa237104395e4a4438af50cbL4-R8) [[3]](diffhunk://#diff-3c49ded39aba3ab23c6fa84f595084cdcf5d219729912f355a7eb24895cbfeeaL4-R8) [[4]](diffhunk://#diff-74da782083bbd46f709bd607809c5d9f9fe4294a93679382cb722d3e22762067L11-R18) [[5]](diffhunk://#diff-6e6cad8f56c0f16661dbb2df163995bd9d9b297b422009e906d024a2722361fcL497-R497) [[6]](diffhunk://#diff-6e6cad8f56c0f16661dbb2df163995bd9d9b297b422009e906d024a2722361fcL555-R555) [[7]](diffhunk://#diff-6e6cad8f56c0f16661dbb2df163995bd9d9b297b422009e906d024a2722361fcL587-R587) [[8]](diffhunk://#diff-6e6cad8f56c0f16661dbb2df163995bd9d9b297b422009e906d024a2722361fcL621-R621)

### Documentation Updates:
* Updated the system requirements and WebRTC version badge in `README.md` to reflect the new iOS version and WebRTC version. [[1]](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L3-R3) [[2]](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L21-R21)

### Checksum Update:
* Updated the checksum for the new WebRTC binary target in `Package.swift`.

### Change Log:
* Added entries to `CHANGES.md` documenting the WebRTC version upgrade and iOS deployment target update.

以下は過去の PR
---

- [UPDATE] WebRTC m129.6668.1.0 に上げる
  - @miosakuma
- [UPDATE] IPHONEOS_DEPLOYMENT_TARGET を 14.0 に上げる
  - libwebrtc の変更への追従
    - https://webrtc.googlesource.com/src/+/9b81d2c954128831c62d8a0657c7f955b3c02d32

SwiftPM、CocoaPods を利用してサンプルアプリがビルドされることを確認済です。

---
This pull request includes several updates to the iOS deployment target and WebRTC framework version across multiple files. The main changes involve upgrading the WebRTC framework to version `129.6668.1.0` and raising the iOS deployment target to `14.0`.

### WebRTC Framework Update:
* Updated WebRTC framework version to `129.6668.1.0` in `Package.swift`, `Podfile`, `Podfile.dev`, `Sora.podspec`, and `README.md`. [[1]](diffhunk://#diff-f913940c58e8744a2af1c68b909bb6383e49007e6c5a12fb03104a9006ae677eL6-R6) [[2]](diffhunk://#diff-8f7d6adf31268a2d897ee34bd170592648d6e520aa237104395e4a4438af50cbL4-R8) [[3]](diffhunk://#diff-3c49ded39aba3ab23c6fa84f595084cdcf5d219729912f355a7eb24895cbfeeaL4-R8) [[4]](diffhunk://#diff-74da782083bbd46f709bd607809c5d9f9fe4294a93679382cb722d3e22762067L11-R18) [[5]](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L3-R3)

### iOS Deployment Target Update:
* Raised iOS deployment target from `13.0` to `14.0` in `Podfile`, `Podfile.dev`, `Sora.podspec`, and `Sora.xcodeproj/project.pbxproj`. [[1]](diffhunk://#diff-8f7d6adf31268a2d897ee34bd170592648d6e520aa237104395e4a4438af50cbL4-R8) [[2]](diffhunk://#diff-3c49ded39aba3ab23c6fa84f595084cdcf5d219729912f355a7eb24895cbfeeaL4-R8) [[3]](diffhunk://#diff-74da782083bbd46f709bd607809c5d9f9fe4294a93679382cb722d3e22762067L11-R18) [[4]](diffhunk://#diff-6e6cad8f56c0f16661dbb2df163995bd9d9b297b422009e906d024a2722361fcL497-R497) [[5]](diffhunk://#diff-6e6cad8f56c0f16661dbb2df163995bd9d9b297b422009e906d024a2722361fcL555-R555) [[6]](diffhunk://#diff-6e6cad8f56c0f16661dbb2df163995bd9d9b297b422009e906d024a2722361fcL587-R587) [[7]](diffhunk://#diff-6e6cad8f56c0f16661dbb2df163995bd9d9b297b422009e906d024a2722361fcL621-R621)

### Documentation Update:
* Updated `CHANGES.md` to reflect the WebRTC framework and iOS deployment target changes.
* Updated `README.md` to reflect the new WebRTC version and iOS deployment target.

### Code Update:
* Updated `Sora/PackageInfo.swift` to reflect the new WebRTC version and branch information.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -11,6 +11,13 @@
 
 ## develop
 
+- [UPDATE] WebRTC m129.6668.1.0 に上げる
+  - @miosakuma
+- [UPDATE] IPHONEOS_DEPLOYMENT_TARGET を 14.0 に上げる
```
Comment (torikizi) [inline CHANGES.md]: 対象とする iOS のバージョンが変わったことがわかりやすい方が良さそうです。

例えば
```
- システム条件の iOS を 14.0 に上げる
  - IPHONEOS_DEPLOYMENT_TARGET を 14.0 に上げる
  - libwebrtc の対象バージョンに追従
    - https://webrtc.googlesource.com/src/+/9b81d2c954128831c62d8a0657c7f955b3c02d32
```

**Review by zztkm — COMMENTED** [review]
UPDATE: URLSessionWebSocketTask が iOS 13.0+ なので、それに合わせて定義されているということを理解できたので、問題ありません

~~@miosakuma URLSessionWebSocketChannel に `@available(iOS 13, *)` とありますが、ここは変更不要でしょうか？~~

https://github.com/shiguredo/sora-ios-sdk/blob/develop/Sora/URLSessionWebSocketChannel.swift#L3

File: CHANGES.md (hunk 1 above)
Comment (miosakuma) [reply CHANGES.md]: ありがとうございます、提案内容を使わせていただきました。
501a573e で対応しています。

Comment (torikizi) [reply CHANGES.md]: 修正ありがとうございます！

**Review by torikizi — APPROVED** [review]

**Review by zztkm — APPROVED** [review]

**Review by torikizi — APPROVED** [review]
修正ありがとうございます。 +1 です。

File: CHANGES.md [hunk 2]
```diff
@@ -15,6 +15,8 @@
   - @miosakuma
 - [UPDATE] システム条件の iOS を 14.0 に上げる
   - IPHONEOS_DEPLOYMENT_TARGET を 14.0 に上げる
+  - SwiftPM の platforms の設定を 14.0 に上げる
```
Comment (zztkm) [inline CHANGES.md]: 14.0 ではなく、表記に対して素直に `SwiftPM の platforms iOS の設定を v14 に上げる` とかはどうでしょうか？

Comment (miosakuma) [reply CHANGES.md]: ありがとうございます。提案いただいた通りに 988634e0 で対応しました。

**Review by zztkm — APPROVED** [review]

**Pull request merged**